proof-of-spacetime.md

```diff
@@ -112,22 +119,30 @@ Seed [32]byte
 
 ```go
 type Challenge struct {
-    Sector Uint
+    Sector Uint // an index into a list of sectors in a miner's proving set
     Leaf Uint
 }
 ```
 
+```go
+type ReplicaInfo struct {
+    Commitment [32]byte // CommR as produced by the sealing process
+    SectorID   SectorID // identity for a miner's sector
+    Path       string   // used to access the replica's contents when generating a PoSt (i.e. not required for verification)
+}
```

Comment:
> I think from this point of view, paths are an internal detail of the FPS + Sector Base that shouldn't be exposed as such to the abstract caller of GeneratePoSt.

OK. I think it's worth calling out that whomever calls `GeneratePoSt` needs to provide access to the replicas in the proving set. The caller of `VerifyPoSt` does not. 

A `SectorSet` does not provide the required information, and is thus the wrong type. If `SectorSet` was meant to be interpreted in some more-abstract way (i.e. not using the `SectorSet` type defined in this specification), then I suggest a new pseudocode type be used. What do you propose? We could clearly use some help here.

Reply:
This api is not meant to include all details, like auxiliary information such as where sectors are stored. The assumption is that this is a private input, and as such the "proof system" which handles these calls has access to them. So I think we should stick to passing in a `SectorSet`, and the mapping of such set, such that it matches to the challenges happens in the function calls of `Generate/VerifyPoSt`, rather than outside.